docs/_docs/reference/contextual/derivation.md

```diff
@@ -419,7 +419,7 @@ object Eq:
           case ((x, y), elem) => check(elem)(x, y)
         }
 
-  inline given derived[T](using m: Mirror.Of[T]): Eq[T] =
+  inline def derived[T](using m: Mirror.Of[T]): Eq[T] =
```

Comment:
I believe there is a test case with this example that also needs to be updated.

Reply:
a cursory search of "given derived" reveals quite a lot of instances, including in the `docs/_docs/reference/contextual/derivation-macro.md` document, which is derivation using macro quotes and splices instead of inline match